buildenv/jenkins/common/build.groovy

```diff
@@ -673,12 +673,16 @@ def upload_artifactory_core(geo, uploadSpec) {
     // Add BUILD_IDENTIFIER to the buildInfo. The UploadSpec adds it to the Artifact info
     buildInfo.env.filter.addInclude("BUILD_IDENTIFIER")
     buildInfo.env.capture = true
+    if (ARTIFACTORY_CONFIG[geo]['buildNamePrefix'] != '') {
+        buildInfo.name = ARTIFACTORY_CONFIG[geo]['buildNamePrefix'] + '/' + JOB_NAME
+    }
+    println "buildinfo.name:$buildInfo.name"
```

Comment:
If this is intended to remain, please fix the typo (`buildinfo` should be `buildInfo`).

Reply:
fixed